src/integrations/admin/indexing-notification-integration.php

```diff
@@ -223,6 +243,7 @@ protected function notification() {
 			[
 				'type'         => Yoast_Notification::WARNING,
 				'id'           => self::NOTIFICATION_ID,
+				'data_json'    => [ 'reason' => $reason ],
```

Comment:
I don't think we should compare with a reason. Because the switch statement to declare a reason has a default case. 

It might be better to have a method on notification that gets the message (content) something like `get_message`